go/vt/mysqlctl/s3backupstorage/s3.go

```diff
@@ -496,6 +497,19 @@ func objName(parts ...string) *string {
 	return &res
 }
 
+// This creates a new transport based off http.DefaultTransport the first time and returns the same
+// transport on subsequent calls so connections can be reused as part of the same transport.
+func getS3Transport() *http.Transport {
+	if defaultS3Transport == nil {
+		tlsClientConf := &tls.Config{InsecureSkipVerify: tlsSkipVerifyCert}
+
+		defaultS3Transport = http.DefaultTransport.(*http.Transport).Clone()
```

Comment:
This seems correct after reading through: https://github.com/golang/go/issues/26013

Reply:
I think that we should get rid of this getter and instead initialize the variable in `init()` though. This is a more clear signal of our intent and a more standard pattern. Does that make sense?

Reply:
yes, that makes sense. I will move it to the `init()` function. I will updated the PR tomorrow